src/billing/components/Free/PAYGConversion.tsx

```diff
@@ -12,8 +12,64 @@ import {
   Button,
 } from '@influxdata/clockface'
 import {useHistory} from 'react-router-dom'
+import CloudUpgradeButton from 'src/shared/components/CloudUpgradeButton'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import {GoogleOptimizeExperiment} from 'src/cloud/components/experiments/GoogleOptimizeExperiment'
+import {CREDIT_250_EXPERIMENT_ID} from 'src/shared/constants'
 
-const PAYGConversion: FC = () => {
+export const Credit250PAYGConversion: FC = () => {
+  if (isFlagEnabled('credit250Experiment')) {
+    return (
+      <GoogleOptimizeExperiment
+        experimentID={CREDIT_250_EXPERIMENT_ID}
+        variants={[
+          <Grid className="credit-250-conversion-panel" key="1">
+            <Grid.Column
+              className="credit-250-conversion-panel--message"
+              widthSM={Columns.Four}
+            >
+              <Heading
+                className="credit-250-conversion-panel--header"
+                element={HeadingElement.H3}
+                weight={FontWeight.Bold}
+              >
+                Get free $250 credit
+              </Heading>
+              <Heading
+                className="credit-250-conversion-panel--header"
+                element={HeadingElement.H5}
+                weight={FontWeight.Regular}
+              >
+                Credit applied to first 30 days of usage
+              </Heading>
+              <CloudUpgradeButton
+                className="credit-250-conversion-upgrade--button"
+                showPromoMessage={false}
+                size={ComponentSize.Large}
+              />
+            </Grid.Column>
+            <Grid.Column widthSM={Columns.Eight}>
+              <div className="conversion-panel--list">
+                <ul className="credit-250-conversion-panel--benefits">
+                  <li>Unlimited buckets to store your data</li>
+                  <li>Unlimited storage retention</li>
+                  <li>Unlimited dashboards</li>
+                  <li>Unlimited tasks</li>
+                  <li>Unlimited alert checks and notification rules</li>
+                  <li>HTTP and PagerDuty notifications</li>
+                  <li>Up to 1,000,000 series cardinality</li>
```

Comment:
Let's wrap this in a `new Intl.NumberFormat()`. I'm not sure if we have requirements that this be formatted in American english, but even if we hard code the locale to `en-US`, this will give us a bit of future proofing if we ever decide to go international. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/format.

Something like

```ts
const numberFormatter = new Intl.NumberFormat('en-US')
...
<li>Up to {numberFormatter.format(1000000)} series cardinality</li>
```